xDrip/Components/Charts/SupportingLogic/GlucoseChartProvider.swift

```diff
@@ -46,5 +71,175 @@ extension GlucoseChartProvider where Self: UIView {
             context.setFillColor(color.cgColor)
             context.fillEllipse(in: circleRect)
         }
+        
+        drawBasalStroke()
```

Comment:
Call this method from the outside, because it might not be required for other cases (like history).